includes/class-amp-story-post-type.php

```diff
@@ -105,6 +105,39 @@ public static function register() {
 			]
 		);
 
+		$meta_args = [
+			'type'           => 'array',
+			'object_subtype' => self::POST_TYPE_SLUG,
+			'single'         => true,
+			'show_in_rest'   => [
+				'schema' => [
+					'type'  => 'array',
+					'items' => [
+						'id'       => [
+							'type' => 'string',
+						],
+						'type'     => [
+							'type'    => 'string',
+							'default' => 'page',
+						],
+						'elements' => [
+							'type'    => 'array',
+							'default' => [],
+						],
+						'index'    => [
+							'type'    => 'integer',
+							'default' => 0,
+						],
+					],
+				],
+			],
+		];
+		// Hide the pages data from none logged in users.
+		if ( ! is_user_logged_in() ) {
+			$meta_args['show_in_rest'] = false;
```

Comment:
Couldn't this always be `true` but just use an `auth_callback`? ([docs](https://developer.wordpress.org/reference/functions/register_meta/#parameters))

Reply:
`auth_callback` is just for writes and not reads. See the docs 
> (string) Optional. A function or method to call when performing edit_post_meta, add_post_meta, and delete_post_meta capability checks.`